recipes/vertica_highcharts/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "vertica_highcharts" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vertica-highcharts-{{ version }}.tar.gz
+  sha256: 50a4a5b9daadebbf65b2e62e9e9f456d0edea049bb252e37369e35cfc56948ad
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - jinja2
+    - future
+
+test:
+  imports:
+    - vertica_highcharts
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/vertica/vertica-highcharts
```

Comment:
It looks like this URL doesn't exist

Reply:
it's a private repo, that's normal. We can make it public if needed.

Reply:
That seems a little strange given the recipe says this is under an open source license (MIT). Why is it private?

Reply:
it is the fork of another project and we've modified some part of it to fit our goals as it was not maintained by the original developers. Is it a problem to keep it private?

Reply:
@afard please look at the conversation and advise.

Reply:
I don't see why not making the vertica-highcharts repository public.
@oualib, I assume you know the procedure of making a repository in Vertica organization public.

Reply:
@afard will do.